presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -59,7 +63,7 @@ Property Name                   Description
 ``kafka.buffer-size``           Kafka read buffer size
 ``kafka.table-description-dir`` Directory containing topic description files
 ``kafka.hide-internal-columns`` Controls whether internal columns are part of the table schema or not
-``kafka.messages-per-split``    Number of messages that will be processed by single Presto split
+``kafka.messages-per-split``    Number of messages that are processed by each Presto split, defaults to 100000, smaller numbers result in higher parallelization of the query processing
```

Comment:
I am not sure if we should document defaults. It could change, causing docs to be out-of-date. This is general problem, but hardcoding values is like crossing the border to me.

> , smaller numbers result in higher parallelization of the query processing

I would remove that. This sentence suggest that the best configuration is 1, while in reality it is a tread off (you don't mention the other force). My thought is that, it is not trivial, folks who understand that don't need a comment here, folks that don't understand this could be easily mislead.

Reply:
We document defaults all over the place .. but I am happy to leave it out here and also leave out the complexity. The only problem then is if someone wants to tune .. how do they know what the start value is..  so I am torn. Wdyt?

Reply:
I don't know what we should do here. I just shared my concerns.

Reply:
How about just documenting the value but remove the "smaller numbers..." section since it too simplistic?

Reply:
@kokosing  @electrum ?

Reply:
Sounds good to me.

Reply:
Done